src/probnum/diffeq/__init__.py

```diff
@@ -11,7 +11,14 @@
     threebody,
     vanderpol,
 )
-from .odefiltsmooth import GaussianIVPFilter, ivp2ekf0, ivp2ekf1, ivp2ukf, probsolve_ivp
+from .odefiltsmooth import (
+    GaussianIVPFilter,
+    KalmanODESolution,
```

Comment:
I am naming it `KalmanODESolution` to emphasise that it does not really depend on the ODE filter anymore, but behaves like any Gauss-Markov posterior.

Reply:
also, I changed the diffeq-internal imports to relative imports (I think this is better wrt circular imports, etc.)